website/source/api/agent/connect.html.md

```diff
@@ -259,7 +259,7 @@ blocking](/api/index.html#hash-based-blocking-queries) only.
 
 ```text
 $ curl \
-   http://127.0.0.1:8500/v1/connect/proxy/web-proxy
+   http://127.0.0.1:8500/v1/agent/connect/proxy/web-proxy
```

Comment:
This is a legit fix that would be good to merge, thanks!

Reply:
Authorship really doesn't worry me Paul, please go ahead and just make the change and I'll close the PR.